lib/rubocop/cop/layout/indent_array.rb

```diff
@@ -20,34 +20,65 @@ module Layout
       # styles are 'consistent' and 'align_brackets'. Here are examples:
       #
       # @example EnforcedStyle: special_inside_parentheses
+      #   # The `special_inside_parentheses` style enforces that the first
+      #   # element in an array literal where the opening bracket and first
+      #   # element are on seprate lines is indented one step (two spaces) more
+      #   # than the position inside the opening parenthesis.
       #
-      #   # special_inside_parentheses
+      #   #bad
+      #   array = [
+      #     :value
+      #   ]
+      #   and_in_a_method_call([
+      #     :no_difference
+      #                        ])
+      #
+      #   #good
       #   array = [
       #     :value
       #   ]
```

Comment:
It looks like the `good` and `bad` example for `array = [:value]` are identical. Am I missing something, or does this need a little patch?

Reply:
In the good example, the element in the method call is indented one step more than the opening brackets, in the bad example it's indented from the method definition instead. Good example of the value of linters :P

Reply:
I see the difference between the examples in lines 32-34 vs 40-42. They look great!

But I still don't see a difference in the example in lines 29-31 vs 37-39 for the array assignment.  Same with lines 52-54 vs 60-62. Should these be removed since they don't highlight a difference?